.circleci/config.yml

```diff
@@ -374,7 +374,7 @@ workflows:
           matrix: { parameters: { rails: ['7.0', '7.1'], ruby: ['3.0'], database: ['mysql'], paperclip: [true] } }
       - test_solidus:
           name: *name
-          matrix: { parameters: { rails: ['7.0', '7.1'], ruby: ['3.1'], database: ['postgres'], paperclip: [false] } }
```

Comment:
Do you know why we are not getting the same error on https://github.com/solidusio/solidus/pull/5788/files#diff-78a8a19706dbd2a4425dd72bdab0502ed7a2cef16365ab7030a5a0588927bf47L383?

Reply:
No

Reply:
Maybe because 3.3.2 has the particular issue https://github.com/solidusio/solidus/pull/5793#issuecomment-2167518956

Reply:
I rebased after we merged #5793